src/sparsezoo/objects/file.py

```diff
@@ -60,7 +60,7 @@ def __init__(
         # has a local path attached to it. Therefore,
         # either has been downloaded or can be
         # downloaded
-        self._path = path
+        self.path = path
         self.owner_path = owner_path
```

Comment:
just noticed - can we get a better name for owner path? is this not always relative to path / can we call it parent directory?

Reply:
sounds good